alpha/lib/model/DeliveryProfileLive.php

```diff
@@ -462,5 +462,18 @@ public function getLivePackagerSigningDomain()
 	{
 		return $this->getFromCustomData("livePackagerSigningDomain");
 	}
+
+	public function setShouldRedirect($v)
+	{
+		$this->shouldRedirect = $v;
```

Comment:
why add this line. doesn't it raise an error since you removed the parameter from the object?

Reply:
As far as I see, the "shouldRedirect" is still on the obj - so no error.
However - I not sure we want to set this when calling "setShouldRedirect" - this function will use for setting the default state and the actual property will be change when it is dynamically done during the playManifest request.
Due to the fact you didn't use this function in this PR I guess that was the original intention - so this line still need to be remove in my opinion
The other way to do it is define new property and use him but I not sure we want to go there

Reply:
My first intention was to avoid case that someone in the future will call "setShouldRedirect" from the "playManifestAction" for example, and expect that the internal parameter will be changed.
It can be removed but it might cause misunderstandings in the future

Reply:
anyway - updated the PR, the setter won't affect the internal "shouldRedirect" anymore. added comment to the setter